server/src/server.ts

```diff
@@ -418,10 +420,14 @@ connection.onImplementation((pos: TextDocumentPositionParams) => {
 });
 
 // This handler provides the initial list of the completion items.
-connection.onCompletion((pos: TextDocumentPositionParams): CompletionItem[] => {
+connection.onCompletion((pos: CompletionParams): CompletionItem[] => {
     const doc: TextDocument = documents.get(pos.textDocument.uri);
     return completionProvider.getAvailableCompletionItems(pos.textDocument.uri, doc.offsetAt(pos.position));
 });
 
+// When the event onCompletion occurs, we send to the client light version of AbstractYmlObject.
```

Comment:
Not sure, not clear, two suggs.

```suggestion
// When the event onCompletion occurs, we send to the client a light version of each AbstractYmlObject.
```

```suggestion
// When the event onCompletion occurs, we send to the client a light version of the relevant AbstractYmlObject.
```